src/main/res/values/strings.xml

```diff
@@ -60,4 +60,6 @@
     <string name="default_counter_name">First counter</string>
     <string name="unknown">Unknown</string>
 
+    <string name="dialog_edit_max">Maximum</string>
```

Comment:
Where did this string come from? Why is it necessary?

Reply:
Sorry, that  is added for exercise but unintentionally committed, do I need push it again?

Reply:
Yes, please update the CR to remove it.